proxy/proxy.go

```diff
@@ -133,3 +133,29 @@ func (s *Settings) FullNoProxy() string {
 	noProxySet := set.NewStrings(allNoProxy...)
 	return strings.Join(noProxySet.SortedValues(), ",")
 }
+
+// FullNoProxy merges NoProxy and AutoNoProxyList
```

Comment:
not sure what happened here, but these two are copies of the function above - please drop the dupes